api/applications/views.py

```diff
@@ -50,7 +50,7 @@ class ApplicationList(JSONAPIBaseView, generics.ListCreateAPIView, ODMFilterMixi
     view_category = 'applications'
     view_name = 'application-list'
 
-    renderer_classes = [renderers.JSONRenderer]  # Hide from web-browsable API tool
+    renderer_classes = [JSONRendererWithESISupport, JSONAPIRenderer, ]  # Hide from web-browsable API tool
```

Comment:
Our real goal was to keep this an undocumented (private) API not intended for external use- hence the visual difference. But swagger has a different mechanism for excluding things.

I'm ok with removing this if we switch to swagger, as the current mechanism of hiding things is pretty weak anyway. (and creates varnish maintenance headaches) Usage will still be controlled separately, by permissions classes.

Can you please add a TODO or somesuch, so that we remember to hide the endpoint in a future doc revamp?